src/actions/checkCoupon.js

```diff
@@ -0,0 +1,19 @@
+import { createActions } from 'redux-actions';
+import { ax, CHECK_COUPON_URL, authHeader } from '../util/api';
+import deauthenticateIfNeeded from './deauthenticateIfNeeded';
+
+export const COUPON_CHECK_REQUEST = 'COUPON_CHECK_REQUEST';
```

Comment:
Possibly comment what these variables do.

Reply:
Massive commenting of everything will happen later!